federation/apis/federation/register.go

```diff
@@ -48,7 +48,6 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&Cluster{},
 		&ClusterList{},
 		&api.ListOptions{},
-		&api.DeleteOptions{},
```

Comment:
Yes, we should let this be here.

> I found there is no such option in other register.go files.

Can you point me to the register.go file you are referring to. I see that DeleteOptions is included in our core register.go: https://github.com/kubernetes/kubernetes/blob/ada5669c7f67fbe2108914cf91476ca944e4cc05/pkg/api/register.go#L115